src/test/java/core/basesyntax/UserTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserTest {
+    private RegistrationServiceImpl registration;
+    private User user;
+    private StorageDao storageDaoImpl;
+
+    @BeforeEach
+    void setUp() {
+        registration = new RegistrationServiceImpl();
+        storageDaoImpl = new StorageDaoImpl();
+        user = new User();
+        user.setLogin("Ivan Zhuravlev");
+        user.setAge(18);
+        user.setPassword("my password 123");
+    }
+
+    @Test
+    void register_Ok() {
```

Comment:
```suggestion
    void register_user_isOk() {
```
rewrite all test names